src/components/Sidebar/SidebarPluginSection/SidebarPluginSection.tsx

```
@@ -0,0 +1,139 @@
+          onClick={onItemClick}
+          className={({ isActive }) =>
+            useSimpleButton
+              ? isActive
+                ? styles.leftDrawerActiveButton
+                : styles.leftDrawerInactiveButton
+              : isActive
+                ? styles.sidebarBtnActive
+                : styles.sidebarBtn
+          }
+          data-testid={`plugin-${item.pluginId}-btn`}
+          role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)